packages/ui/internal/localized-fonts/localized-fonts.js

```diff
@@ -53,13 +53,19 @@ const buildFontSet = function (fontName, fonts, strLang, bitDefault) {
 	return strOut;
 };
 
-function fontGenerator (locale) {
+function fontGenerator (locale, localeChanged) {
+	const styleId = 'localized-fonts';
+	let styleElem = document.getElementById(styleId);
```

Comment:
A `if (typeof document !== 'undefined') {` guard is required for `document` usage so it doesn't error on prerendering in Node environment.